paddle/fluid/platform/mkldnn_reuse.h

```diff
@@ -938,11 +938,34 @@ class ActivationMKLDNNHandler
       beta = ctx.HasInput("Max") ? ctx.Input<Tensor>("Max")->data<float>()[0]
                                  : ctx.Attr<float>("max");
     } else {
-      // paddle uses beta but mkldnn uses alpha for swish
-      if (algorithm == dnnl::algorithm::eltwise_swish) {
-        std::swap(alpha, beta);
-      } else if (algorithm == dnnl::algorithm::eltwise_bounded_relu) {
-        alpha = ctx.Attr<float>("threshold");
+      switch (algorithm) {
+        // alpha and beta does not exist for those
+        case dnnl::algorithm::eltwise_abs:
+        case dnnl::algorithm::eltwise_exp:
+        case dnnl::algorithm::eltwise_gelu:
+        case dnnl::algorithm::eltwise_hardswish:
+        case dnnl::algorithm::eltwise_logistic:
+        case dnnl::algorithm::eltwise_mish:
+        case dnnl::algorithm::eltwise_relu:
+        case dnnl::algorithm::eltwise_sqrt:
+        case dnnl::algorithm::eltwise_tanh:
+          break;
+        // alpha is defined for following ones
+        case dnnl::algorithm::eltwise_elu:
+        case dnnl::algorithm::eltwise_leaky_relu:
+          alpha = ctx.Attr<float>("alpha");
+          break;
+        case dnnl::algorithm::eltwise_swish:
+          // paddle uses beta but mkldnn uses alpha for swish
+          alpha = ctx.Attr<float>("beta");
+          break;
+        case dnnl::algorithm::eltwise_bounded_relu:
+          alpha = ctx.Attr<float>("threshold");
+          break;
+        default:
+          alpha = ctx.HasAttr("alpha") ? ctx.Attr<float>("alpha") : 0;
```

Comment:
I have checked and after including 3 more eltwise algorithms which are mentioned above `default` branch of this switch-case won't ever be chosen, since all eltwises would be already covered(only `clip` and `linear` are using two arguments and are covered in lines 921-940).